tldr/cfg_extractor.py

```
@@ -2197,3 +2209,303 @@ def extract_elixir_cfg(source: str, function_name: str) -> CFGInfo:
 
     builder = ElixirCFGBuilder(source_bytes)
     return builder.build(func_node, function_name)
+
+
+class ZigCFGBuilder:
+
+    def __init__(self, source: bytes):
```

Ruff (0.14.14)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)